### ISSUE TITLE
[Draft] Referral program

### ISSUE THREAD

**Issue by ripcurlx** [bisq-network/growth#8]
## Learning Goal
Can we sustainable increase growth by setting up a referral program?

## Hypothesis
By incentivising people to invite their friends we expect an additional growth of downloads/trades by x%.

## Metric
By tracking accepted invite codes within the platform we will be able to measure the success and calculate a growth factor for it.

## Experiment
For this program to work we need to think about a referral system that is hard to exploit and has a high incentive for people to invite friends. A incentivation structure could look like following table (inspiration from altcoin.io)

> Refer 10 people - Earn 5% of trading fees from your referrals
> Refer 25 people - Earn 10% of trading fees from your referrals
> Refer 50 people - Earn 15% of trading fees from your referrals
> Refer 100 people - Earn 20% of trading fees from your referrals
> Refer 500 people - Earn 25% of trading fees from your referrals

## Result
> _- What happened?_
> _- What data did we collect?_
> _- Anything unexpected?_

## Next Step
> _- Pivot or persevere?_
> _- Another experiment for this goal?_
> _- Do we need to clean up?_

**Comment by ManfredKarrer** [comment]
What is your idea how to pay out the trade fee? It goes directly to the arbitrator in the trade process. With BSQ fees they will get burned (too small  to transfer in a BTC tx).
I could only see a way to use the compensation requests but they are usually designed for larger payments (min 100 USD) not for micro payments and require a fee in BSQ. They also add work for all stakeholders for evaluating a request at the voting process. So that seems to me also not a good way for using it for a referral program.

Maybe we could use some non-monetary things like gadgets (T-Shirt, Bisq sticker,...)? But then shipping costs are an issue for small value items... Digital gits would be best, but no concrete idea which one (no cryptokittens ;-)).

**Comment by ripcurlx** [comment]
Yes, it is quite tricky in our case to set it up, especially if we don't have "real" user accounts. But I wanted to put it out there, so we, as a crowd, might come up with a good idea on how to nudge people to share Bisq with their friends (besides the fact that Bisq is such a great tool, that everyone wants to share it anyways ;-) ).

**Comment by flix1** [comment]
This is interesting... I have been thinking quite a bit about how to incentivise growth and a referral progarm might work and be scalable. However it is very hard to do without identity..

**Comment by ManfredKarrer** [comment]
"For example a 50 BSQ reward for anyone who does a first trade in a new currency or successfully tests a new payment method.": Good idea!

**Comment by FKrauss** [comment]
Just adding some comments here after being part of referral programs myself and even exploiting some of them: 

- Tiers should be based on trading volume, not users. It is hard for us to validate a user, yet it is easy for a scammer to get 50 users with only the minimum activity required -> they way Bitmex does it good -> up to 10 BTC 10% of trading fees, up to 1000 BTC 15%, more than 10k 20%. See [here](https://www.bitmex.com/app/affiliates)

- There should be an advantage for the user to signup via the referral. (i.e. Bitmex's 10% off trading fees for the first 6 months) Otherwise advocates will have no selling point to offer people other than being thankful for them for having shared info on this cool service.

- Referral program policy that always needs to be in place: search ads in the term bisq are not allowed! - I can go into details in the growth call, but this is the easiest and cheapest way of exploiting a referral program. 50% of people that search for a brand name click the ad and not the organic result. So a referral scammer can buy these ads and get 50% of the organic traffic (which is high converting) and thus be credited to have brought 50% of the best users when in reality he did nothing


Well, let's chat in our next call, I see from the discussion that the first step we need to do here is figure out the mechanics of having such a thing while maintaining user privacy
